.github/workflows/build-php-db-import-export.yml

```diff
@@ -121,6 +119,12 @@ jobs:
         name: Checkout
         uses: actions/checkout@v3
 
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::149899208592:role/ci-storage-backend-BaseStorageBackendRole-5WXOY9DYENCT
+          aws-region: us-east-1
```

Comment:
nemělo by to byt někde ve vars? Buď někde v "hlavičce" nebo ve vars v actions?

Reply:
ve vars v actions zbytecny, je to staticka hodnota. Do hlavicky by to mohlo jit.

Reply:
Tam to nejde definovat do toho main.yml a pak to posielat do jednotlivych build-*.yml filov https://github.com/keboola/storage-backend/actions/runs/4292810716 alebo ako ste to mysleli ?  https://github.com/keboola/storage-backend/pull/45/commits/b83ff5870a27f8d6327497c5f703df24578889aa

Reply:
No to by bylo ideální, ale pomohlo by tomu aj kdybys to měl v tom souboru enom jednůc, takto je tam ten string několikrát staticky vložený.

Reply:
Tak som to upravil dal som to aspon do kazdeho filu na zaciatok c3d69d2a75252d69d03aacad1500e879da1df464